### PR TITLE
Ir 68

### PR DESCRIPTION
**Pull request by attiks**

### CODE REVIEW

File: .docksal/docksal.yml [hunk 1]
```diff
@@ -1 +1,19 @@
 version: "2.1"
+services:
+  # Solr
+  solr:
+    hostname: solr
+    image: docksal/solr:8
```
Comment (cafuego) [inline .docksal/docksal.yml]: Solr 8 won't do. It will have to be 7.7.3.

**Review by cafuego — CHANGES_REQUESTED** [review]
Looks OK except for the solr version, which should probably match what we run on AWS.

File: .docksal/docksal.yml (hunk 1 above)
Comment (attiks) [reply .docksal/docksal.yml]: I'll test locally, but should be fine

Comment (attiks) [reply .docksal/docksal.yml]: Done, config updated

**Pull request merged**